### ISSUE TITLE
Bug with velo example in gmt 6.1.1

### ISSUE THREAD

**Issue by lhoupert** [GenericMappingTools/gmt#4425]
**Description of the problem**

I am currently working in [implementing the velo function in pygmt](https://github.com/GenericMappingTools/pygmt/pull/525).

I just realised that, since gmt 6.1.1, the [first example](https://docs.generic-mapping-tools.org/latest/supplements/geodesy/velo.html#examples) at the end of the `velo` is not giving the same output than in the previous gmt version. Particularly, the current scaling options don't match anymore the description the example:
> The following should make big red arrows with green ellipses, outlined in red. Note that the 39% confidence scaling will give an ellipse which fits inside a rectangle of dimension Esig by Nsig

**Full script that generated the error**

```
gmt velo << END -R-10/10/-10/10 -W0.25p,red -Ggreen -L -Se0.2i/0.39+f18p -B1g1 -Jx0.4/0.4 -A1c+p3p+e -V -pdf test
#Long. Lat. Evel Nvel Esig Nsig CorEN SITE
#(deg) (deg) (mm/yr) (mm/yr)
0. -8. 0.0 0.0 4.0 6.0 0.500 4x6
-8. 5. 3.0 3.0 0.0 0.0 0.500 3x3
0. 0. 4.0 6.0 4.0 6.0 0.500
-5. -5. 6.0 4.0 6.0 4.0 0.500 6x4
5. 0. -6.0 4.0 6.0 4.0 -0.500 -6x4
0. -5. 6.0 -4.0 6.0 4.0 -0.500 6x-4
END
```

**Actual outcome**

![test1](https://user-images.githubusercontent.com/10154151/98383861-ef532280-2044-11eb-9527-3c5eb549eb0b.png)

**Expected outcome**

The expected outcome should be something similar to the figure below:
![test1gmtedit](https://user-images.githubusercontent.com/10154151/98384344-8cae5680-2045-11eb-8c08-cbe6b82ac09a.png)

**System information**

* Operating system: 
ProductName:	Mac OS X
ProductVersion:	10.15.7
BuildVersion:	19H2
* GMT version (`gmt --version`):
6.1.1

**Comment by seisman** [comment]
I just tried the example using GMT 6.1.1, 6.1.0, 6.0.0 and 5.4.5. They all gives me the same results (the **Actual outcome** above). So it seems **velo** has some incompatible change before GMT 5.4.

**Comment by seisman** [comment]
I think this is a problem of the velo example, not a velo bug, and #4426 is a good fix to the velo example.

 @PaulWessel Agree?

**Comment by PaulWessel** [comment]
Yep, seems fine.

**Comment by seisman** [comment]
Closed by #4426.
